ci/Dockerfile

```diff
@@ -1,19 +1,17 @@
-# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+# Copyright (c) 2019-2020 VMware, Inc. All Rights Reserved.
 # SPDX-License-Identifier: BSD-2-Clause
 
-FROM photon:3.0
+FROM debian:buster
 
-# install system dependencies
-# photon:3.0 comes with toybox which conflicts with some dependencies needed for tern to work, so uninstalling
-# toybox first
-RUN tdnf remove -y toybox && tdnf install -y tar findutils attr util-linux python3 python3-pip python3-setuptools git && pip3 install --upgrade pip
+# Install fuse-overlayfs and Tern dependencies
+RUN apt-get update && apt-get -y install wget gnupg2 tar findutils attr util-linux python3 python3-pip python3-setuptools git && pip3 install --upgrade pip && pip3 install tern && echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list && wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_10/Release.key -O Release.key && apt-key add - < Release.key && apt-get update -qq && apt-get -qq -y install buildah fuse-overlayfs && rm -rf /var/cache /var/log/dnf* /var/log/yum.*
 
-# install tern with latest changes
+# Install tern with latest changes
 COPY dist/tern-*.tar.gz .
 RUN pip install tern-*.tar.gz
 
-# make a mounting directory
-RUN mkdir hostmount
+# Adjust storage.conf to enable Fuse storage.
+RUN sed -i -e 's|^#mount_program|mount_program|g' -e '/additionalimage.*/a "/var/lib/shared",' /etc/containers/storage.conf
 
-ENTRYPOINT ["tern", "-q", "-b", "/hostmount"]
```

Comment:
If quiet more is not turned on here, maybe there should be some documentation showing how to turn off logging when running in container mode?

Reply:
Agree. I can add that to do the README.